.circleci/config.yml

```diff
@@ -32,34 +48,96 @@ jobs:
 
       - run:
           name: Add Falcon domain host into /etc/hosts
-          command: echo 127.0.0.1 falcon.docker.localhost | sudo tee -a /etc/hosts
+          command: echo 127.0.0.1 falcon.docker.localhost app.docker.localhost | sudo tee -a /etc/hosts
+
+#      - run:
+#          name: Fix CircleCI files permissons
+#          command: |
+#            sudo chown ubuntu:ubuntu -R *
+#            sudo chown ubuntu:ubuntu -R .[!.]*
+#
+#      - restore_cache:
```

Comment:
@spleshka I believe you will explain me what I did wrong here! :)

Reply:
@PavelBulat given that everything works fine - nothing ;)